docs/UserGuide.md

```diff
@@ -389,25 +389,98 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
+:exclamation: **Caution:** 
+* You cannot undo the sort once you have applied it! 
+* Sort will not affect the list permanently. Trackermon rearranges the show list one time after a successful sort command! 
+* After a successful sort command followed by an add command will still add the new show to the last index!  
 </div>
 
 **Format:** `sort [n/ORDER] [s/ORDER] [t/ORDER] [r/ORDER] [so/SEQUENCE]`
 
+**Explanation** 
+
+Let's say you want to sort by name in ascending order,
+use `sort n/asc`.
+
+Let's say you want to sort by name in descending order,
+use `sort n/dsc`.
+
+<div markdown="block" class="alert alert-warning">
+:exclamation: **Take note** We sort name lexicographically, in other words, we sort it by comparing alphabets and numbers. We do not compare by combining the numbers into a unit. For example, in ascending order, 100Doggy comes before 69Doggy!   
+</div>
+
+
+Let's say you want to sort by status in ascending order,
+use `sort s/asc`. 
+
+Let's say you want to sort by status in descending order,
+use `sort s/dsc`.
+
+<div markdown="block" class="alert alert-warning">
+:exclamation: **Take note** We sort status in this ascending order: `completed`, `watching`, `plan-to-watch`.
+</div>
+
+
+Let's say you want to sort by number of tags in ascending order,
+use `sort t/asc`.
+
+Let's say you want to sort by number of tags in descending order,
+use `sort t/dsc`.
+
+<div markdown="block" class="alert alert-warning">
+:exclamation: **Take note** We sort tags by number of tags and not by comparing the names of tags.
+</div>
+
+Let's say you want to sort by rating in ascending order,
+use `sort r/asc`.
+
+Let's say you want to sort by rating in descending order,
+use `sort r/dsc`.
+
+Let's say you want to sort by name in ascending order then by status in ascending order,
+use `sort n/asc s/asc`.
+
+<div markdown="block" class="alert alert-warning">
+:exclamation: **Take note** The order of how you type the prefixes does not affect the outcome! If you type `sort s/asc n/asc`, Trackermon will still sort by name in ascending order then by status in ascending order
+</div>
+
+Therefore, let's say you want to sort by status in ascending order then by name in ascending order,
+use `sort n/asc s/asc so/statusname`.
+
+<div markdown="block" class="alert alert-warning">
+:exclamation: **Take note** We use `so/SEQUENCE` to swap the priority of the sorting criteria! By default, Trackermon always prioritises sorting in the following order if the criteria are used:
+
+  1. name
+  2. status
+  3. rating
+  4. tag
+
+So, use the "tagratingstatusname" as the `SEQUENCE` if you want to swap the ordering to:
+1. tag
+2. rating
+3. status
+4. name
+
+Additionally, when using `so/` prefix, Trackermon will check if the required criteria is presented in your `SEQUENCE`.
+So if you use `sort n/asc s/asc so/haha`, the required criteria are name and status. As name and status are not spelt in full for `SEQUENCE`, Trackermon will tell you the input is invalid!
```

Comment:
Would `As name and status are not spelt in full` be better replaced by `As name and status are not provided` instead?